### PR TITLE
GetSynopses() moved to trait

### PR DESCRIPTION
**Pull request by tatitati**

### CODE REVIEW

File: src/Mapper/Traits/SynopsesTrait.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Mapper\Traits;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Synopses;
+
+trait SynopsesTrait
+{
+    private function getSynopses(array $dbImage): Synopses
```
Comment (votemike) [inline src/Mapper/Traits/SynopsesTrait.php]: https://gist.github.com/votemike/04d7f5c2859a6999a5aa3915ea40c982#file-arraydocblocks-php-L55

File: src/Mapper/ProgrammesDbToDomain/SegmentMapper.php [hunk 2]
```diff
@@ -5,12 +5,14 @@
...
 use BBC\ProgrammesPagesService\Domain\Entity\Segment;
 use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
-use BBC\ProgrammesPagesService\Domain\ValueObject\Synopses;
+use BBC\ProgrammesPagesService\Mapper\Traits\SynopsesTrait;
 
 class SegmentMapper extends AbstractMapper
 {
     const MUSIC_TYPES = ['music', 'classical'];
 
+    use SynopsesTrait;
```
Comment (BPScott) [inline src/Mapper/ProgrammesDbToDomain/SegmentMapper.php]: Should `use` blocks be before `const` declarations?

File: src/Mapper/Traits/SynopsesTrait.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+namespace BBC\ProgrammesPagesService\Mapper\Traits;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Synopses;
+
+trait SynopsesTrait
+{
+    /**
+     * @param mixed[] $dbImage
+     */
+    private function getSynopses(array $dbImage): Synopses
```
Comment (BPScott) [inline src/Mapper/Traits/SynopsesTrait.php]: `$dbEntity` as it won't always be an image?

**Comment by votemike** [comment]
Does the trait need a test?

**Comment by votemike** [comment]
Looking at the deleted code, the method wasn't always the same, some of them were missing typehints. Good we refactored 👍

**Pull request merged**